packages/tools/client-debugger/client-debugger-view/tsconfig.json

```diff
@@ -7,6 +7,8 @@
 		"target": "es6",
 		"jsx": "react",
 		"types": ["jest"],
+		"skipLibCheck": true,
 	},
+	"exclude": [ "node_modules/@fluentui/react-components/**/*"],
```

Comment:
This shouldn't be needed - the base config we extend already specified `node_modules` under its `exclude` property.

Reply:
Huh... I thought this line was what served as a workaround and let the build succeed 🤔

Reply:
@alexvy86 the part that fixes the issue is `"skipLibCheck": true,`